single.php

```diff
@@ -6,6 +6,7 @@
 $social       = ( shortcode_exists( 'ucf-social-links' ) ) ? do_shortcode( '[ucf-social-links layout="affixed"]' ) : '';
 $author_bio   = today_get_post_author_bio( $post );
 $related      = today_get_post_related( $post );
+$tag_cloud    = today_get_tag_cloud( $post, 'mb-4' );
```

Comment:
Adding `mt-5` or maybe `mt-4`  here would help add vertical space between post content and the tag cloud on stories where a Source isn't set.